e2e/test/scenarios/visualizations-tabular/drillthroughs/column_extract_drill.cy.spec.js

```diff
@@ -230,12 +237,23 @@ describe("extract action", () => {
   });
 });
 
-function extractColumnAndCheck({ column, option, newColumn = option, value }) {
+function extractColumnAndCheck({
+  column,
+  option,
+  newColumn = option,
+  value,
+  example,
+}) {
   const requestAlias = _.uniqueId("dataset");
   cy.intercept("POST", "/api/dataset").as(requestAlias);
   cy.findByRole("columnheader", { name: column }).click();
   popover().findByText("Extract day, month…").click();
   cy.wait(1);
+
+  if (example) {
+    popover().findByText(option).should("contain", example);
+  }
```

Comment:
`contain` can be too inclusive in some cases.
If there's a selector (even a test-id) that wraps an example, it would be a tiny bit safer to assert `cy.get(selector).should("have.text", example)`

Something along those lines.